documentation/src/main/asciidoc/topics/con_cache_encoding.adoc

```diff
@@ -6,19 +6,21 @@ Encoding is the format, identified by a media type, that {brandname} uses to sto
 == Remote caches
 
 {brandname} Server stores entries in remote caches with the encoding that is set in the cache configuration.
-If the remote cache does not have any encoding configuration, {brandname} Server stores keys and values without any media type information.
-Hot Rod and REST clients also use encoding to operate on data and include a media type with each request to {brandname} Server.
 
-If you do not configure encoding for remote caches then {brandname} Server stores keys and values exactly as the client sends them, discarding any media type information.
-When clients then request entries from these caches, {brandname} Server returns keys and values as they are stored in the remote cache and ignores the client's media type.
-When the remote cache has an encoding, other than `application/unknown`, {brandname} Server converts keys and values to and from the client media type with each request.
+Hot Rod and REST clients include a media type with each request they make to {brandname} Server.
+To handle multiple clients making read and write requests with different media types, {brandname} Server converts data on-demand to and from the media type that is set in the cache configuration.
+
+If the remote cache does not have any encoding configuration, {brandname} Server stores keys and values as generic `byte[]` without any media type information.
+In this case {brandname} Server returns keys and values for any client requests as they are stored in the remote cache, without any media type conversion.
+
+.Use ProtoStream encoding
+
+{brandname} Server cannot convert data to and from certain media types and generates an error when client requests include a media type that is not supported.
```

Comment:
I would focus on request+cache media type pairs, because whether the server can process a request or not depends on both. E.g.

```suggestion
{brandname} Server returns an error when the client request includes a media type that cannot be converted to or from the cache encoding's media type.
```